configs/DURAL/BoardConfig.h

```diff
@@ -26,6 +26,9 @@
 #define PIN_BUTTON_R3   0
 #define PIN_BUTTON_A1   3
 #define PIN_BUTTON_A2   2
+#define PIN_BUTTON_TURBO -1
```

Comment:
Generally I am of the thought that we should remove the additional configurations in .44.  

We cannot guarantee compatibility until we have tested the code on those specific devices.  I would not want to push an update out and have a bunch of people say their stuff is not working.

I will reach out to the various project owners.